test/HashBase.test.ts

```diff
@@ -36,59 +47,94 @@ describe('HashBase baselet', function () {
     expect(await getConfig(memlet, dbName)).eql(expectedTest)
   })
   it('insert data', async function () {
-    for (let i = 0; i < dataSet.length; i++) {
-      const data = dataSet[i]
-      await hashbaseDb.insert(partitionName, data.hash, data)
+    for (const partitionName in testDataFixture) {
```

Comment:
Never use `for in`. Always use `for of Object.keys()` instead, which is a safer drop-in replacement.